docs/integrations/sources/gitlab-migrations.md

```diff
@@ -1,8 +1,13 @@
 # Gitlab Migration Guide
 
-## Upgrading to 2.0.0
+## Upgrading to 3.0.0
```

Comment:
please upgrade this with the language above and follow the migration guide template: 
https://docs.google.com/document/d/1V3ybA24Lj_fy7IQwsVVzSxs46qGOsUydwlAQEYSQfcE/edit?pli=1